README.md

```diff
@@ -5,6 +5,8 @@
 
 # README
 
+Heroku link: http://bsfc.herokuapp.com
```

Comment:
We should add more detail to our readme in a future pull request. At the very least, delete the boilerplate.

Reply:
I deleted the boilerplate for now.